CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 ##### Enhancements
 
+* [9621](https://github.com/grafana/loki/pull/9621) **DylanGuedes**: Introduces TSDB postings cache.
```

Comment:
IMO imperative should be used for changelog entries, such as "Introduce", "Add", "Fix", ...

```suggestion
* [9621](https://github.com/grafana/loki/pull/9621) **DylanGuedes**: Introduce TSDB postings cache.
```